src/main/scala/nl.knaw.dans.easy.solr4files/CommandLineOptions.scala

```diff
@@ -39,14 +45,39 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
        |
        |Options:
        |""".stripMargin)
-  //val url = opt[String]("someOption", noshort = true, descr = "Description of the option", default = app.someProperty)
 
+  private val defaultBagStore = Some(configuration.properties.getString("default.bag-store", "MISSING_BAG_STORE"))
+
+  val update = new Subcommand("update") {
+    descr("Update a bag in the SOLR index")
+    val bagStore: ScallopOption[StoreName] = opt[StoreName](
+      "bag-store",
+      default = defaultBagStore,
+      short = 's',
+      descr = "Name of the bag store")
+    val bagUuid: ScallopOption[UUID] = trailArg(name = "bag-uuid", required = true)
+    footer(SUBCOMMAND_SEPARATOR)
+  }
+  val delete = new Subcommand("delete") {
+    descr("Delete all file documents of a bag from the SOLR index")
+    val bagUuid: ScallopOption[UUID] = trailArg(name = "bag-uuid", required = true)
+    footer(SUBCOMMAND_SEPARATOR)
+  }
+  val init = new Subcommand("init") {
```

Comment:
De we need this, you can do delete and update?

Reply:
Update does a single bag, this one all bags.